ci-operator/step-registry/cluster-network-operator/ovn/secondary-encap/validate/cluster-network-operator-ovn-secondary-encap-validate-commands.sh

```
@@ -0,0 +1,112 @@
+    if [[ "${peer_node}" == "${node_name}" ]]; then
+      continue
+    fi
+
+    expected_geneve_peers=$((expected_geneve_peers + 1))
+    if ! grep -Fq "remote_ip=\"${peer_ip}\"" "${show_file}"; then
+      echo "missing geneve remote_ip ${peer_ip} in ovs-vsctl show for node ${node_name}" >&2
+      validation_failed=1
+    fi
+  done < "${mapping_file}"
+
+  local_geneve_ports=$(grep -F "local_ip=\"${expected_ip}\"" "${show_file}" | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[style] Consider using 'grep -c' instead of 'grep|wc -l'.

(SC2126)